### PR TITLE
[NETBEANS-491] Fix issue reporting

### PR DESCRIPTION
**Pull request by rtaneja1**
1. Updated URL in the exception report message - https://issues.apache.org/jira/projects/NETBEANS/issues
2. Review and Report Problem button opens https://issues.apache.org/jira/projects/NETBEANS,
as suggested by Geertjan.

### CODE REVIEW

**Comment by mcdonnell-john** [comment]
Closed PR477 as these are the same now.

**Comment by geertjanw** [comment]
OK, merging this.

**Comment by neilcsmith-net** [comment]
I'd have thought we might be better having a link to the website, from which we can redirect issues as we want? eg. something like https://netbeans.apache.org/ide-report-issue ?  We could set that up as a redirect to JIRA now without hardcoding it into the IDE, giving us the ability to change it easily later.

**Comment by geertjanw** [comment]
I don't see https://issues.apache.org/jira/projects/NETBEANS/issues changing...

**Comment by neilcsmith-net** [comment]
There was talk elsewhere of providing a more user friendly gateway to reporting issues, providing a way for people to report issues without a JIRA account, etc.

**Comment by mcdonnell-john** [comment]
Maybe what we could do in future is have a page on the website similar to https://netbeans.org/community/issues.html that provides information, search functionality etc... and redirects to JIRA.

File: uihandler/src/org/netbeans/modules/uihandler/UIHandler.java [hunk 1]
```diff
@@ -245,8 +249,12 @@ public void actionPerformed(ActionEvent ev) {
         Window w = SwingUtilities.windowForComponent(c);
         if (w != null) {
             w.dispose();
-        } 
-        Installer.RP.post(this);
+        }
```
Comment (JaroslavTulach) [inline uihandler/src/org/netbeans/modules/uihandler/UIHandler.java]: I don't think this is a correct change as it doesn't follow the specified behavior in https://github.com/apache/incubator-netbeans/blob/master/uihandler/arch.xml

Comment (rtaneja1) [reply uihandler/src/org/netbeans/modules/uihandler/UIHandler.java]: Hi, I ran tests for uihandler, those passed, please let me know which other modules I need to test.

Comment (rtaneja1) [reply uihandler/src/org/netbeans/modules/uihandler/UIHandler.java]: I'm looking into this.

**Pull request merged**